plugins/modules/foreman_operatingsystem.py

```diff
@@ -101,7 +101,11 @@
     type: list
   provisioning_templates:
     description:
-      - list of provisioning templates
+      - list of provisioning templates that are associated with the OS object
+      - Specify a full list of the template names you want to associate
+      - with your OS. For example, ["Kickstart default", "Kickstart default finish", "Kickstart default iPXE", "custom"]
+      - After specifying the template associations, you set the default association in
+      - the foreman_os_default_template module.
```

Comment:
```suggestion
      - the M(foreman_os_default_template) module.
```

this will create a link in the html version of the docs, see https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation